python/pyspark/mllib/classification.py

```diff
@@ -223,6 +284,12 @@ def train(cls, data, iterations=100, initialWeights=None, regParam=0.01, regType
                                update (default: 10).
         :param tolerance:      The convergence tolerance of iterations for
                                L-BFGS (default: 1e-4).
+        :param validateData:   Boolean parameter which indicates if the
+                               algorithm should validate data before training.
+                               (default: True)
+        :param numClasses:     The number of possible outcomes for k classes
```

Comment:
Better phrasing: "The number of possible outcomes for k classes classification problem"
-->
"The number of classes (i.e., outcomes) a label can take"